tests/unit/test_exceptions.py

```diff
@@ -79,6 +79,12 @@ def testCodeInvariants(self):
 
     def testInstantiation(self):
         for class_ in self._getExceptionClasses():
+            # some exceptions are becoming too complicated to instantiate
+            # like the rest of the exceptions; just do them manually
+            if class_ in [
+                    exceptions.RequestValidationFailureException,
+                    exceptions.ResponseValidationFailureException]:
+                continue
```

Comment:
To be clear, you would not be fine with a continue, but you would be fine with this?
```
if class_ in [...]:
 args = oneThing
else:
 args = default
```
That seems like a very similar construct...